opal/mca/event/external/configure.m4

```diff
@@ -136,24 +121,41 @@ AC_DEFUN([MCA_opal_event_external_CONFIG],[
                               [opal_event_external_support=yes],
                               [opal_event_external_support=no])
 
-           # Ensure that this libevent has the symbol
-           # "evthread_set_lock_callbacks", which will only exist if
-           # libevent was configured with thread support.
-           LIBS="$opal_event_external_LDFLAGS $LIBS"
-           AC_CHECK_LIB([event], [evthread_set_lock_callbacks],
-                        [],
-                        [AC_MSG_WARN([External libevent does not have thread support])
-                         AC_MSG_WARN([Open MPI requires libevent to be compiled with])
-                         AC_MSG_WARN([thread support enabled])
-                         AC_MSG_ERROR([Cannot continue])])
-
-           AC_CHECK_LIB([event_pthreads], [evthread_use_pthreads],
-                        [],
-                        [AC_MSG_WARN([External libevent does not have thread support])
-                         AC_MSG_WARN([Open MPI requires libevent to be compiled with])
-                         AC_MSG_WARN([thread support enabled])
-                         AC_MSG_ERROR([Cannot continue])])
-
+           AS_IF([test "$opal_event_external_support" = "yes"],
+                 [# Ensure that this libevent has the symbol
+                  # "evthread_set_lock_callbacks", which will only exist if
+                  # libevent was configured with thread support.
+                  LIBS="$opal_event_external_LDFLAGS $LIBS"
+                  AC_CHECK_LIB([event], [evthread_set_lock_callbacks],
+                               [],
+                               [AC_MSG_WARN([External libevent does not have thread support])
+                                AC_MSG_WARN([Open MPI requires libevent to be compiled with])
+                                AC_MSG_WARN([thread support enabled])
+                                opal_event_external_support=no])])
+
+           AS_IF([test "$opal_event_external_support" = "yes"],
+                 [AC_CHECK_LIB([event_pthreads], [evthread_use_pthreads],
+                               [],
+                               [AC_MSG_WARN([External libevent does not have thread support])
+                                AC_MSG_WARN([Open MPI requires libevent to be compiled with])
+                                AC_MSG_WARN([thread support enabled])
+                                opal_event_external_support=no])])
+
+           AS_IF([test "$opal_event_external_support" = "yes" &&
+                  test -z "$with_libevent"],
+                 [AC_MSG_CHECKING([if external libevent version is 2.0.21 or greater])
+                  AC_COMPILE_IFELSE(
+                      [AC_LANG_PROGRAM([[#include <event.h>]],
+                          [[
+#if _EVENT_NUMERIC_VERSION < 0x02001500
+#error "libevent API version is less than 0x02001500"
+#endif
```

Comment:
This needs another version check to ensure it is >= the internal version of libevent (just like hwloc).

Reply:
the purpose of this test is to ensure this.
(and unlike in `hwloc` we do not explicitly require a minimal external version for `libevent`)